src/core/geometry_type.hpp

```diff
@@ -57,13 +57,13 @@ struct RectPadding {
 	 * Get total horizontal padding of RectPadding.
 	 * @return total horizontal padding.
 	 */
-	inline uint Horizontal() const { return this->left + this->right; }
+	inline constexpr uint Horizontal() const { return this->left + this->right; }
```

Comment:
`[dcl.constexpr], §7.1.5/2 in the C++11 standard): "constexpr functions and constexpr constructors are implicitly inline (7.1.2)`.

Could drop the `inline`. Nitpicking ftw :)

Reply:
Done, there was quite a lot added, so worth it :)